VacaYAY/BusinessLogicLayer/Contracts/IAuthService.cs

```diff
@@ -0,0 +1,21 @@
+﻿using BusinessLogicLayer.Dto.EmployeeDto;
+using Microsoft.AspNetCore.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLogicLayer.Contracts
+{
+    public interface IAuthService
+    {
```

Comment:
Is it necessary to return these complex types here?

Reply:
to be honest, i'm not sure what other approach should i take? usermanager and signinmanager methods return identityresult type and signinresult type, so i don't quite understand

Reply:
Think what you need from this methods.

E.G. When you log in, what should you return? Bool, string, etc? 

I think you return a too much unused data.